src/data/graphics/trainers.h

```diff
@@ -425,21 +425,17 @@ static const union AnimCmd sAnimCmd_Kanto[] =
 
 static const union AnimCmd sAnimCmd_Point_HGSS[] =
 {
-    ANIMCMD_FRAME(3, 9),
-    ANIMCMD_FRAME(0, 9),
-    ANIMCMD_FRAME(2, 24),
+    ANIMCMD_FRAME(0, 31),
     ANIMCMD_FRAME(2, 24),
-    ANIMCMD_FRAME(3, 50),
+    ANIMCMD_FRAME(3, 1),
     ANIMCMD_END,
 };
 
 static const union AnimCmd sAnimCmd_Point_HGSS_Red_Leaf[] =
 {
-    ANIMCMD_FRAME(0, 9),
-    ANIMCMD_FRAME(1, 9),
-    ANIMCMD_FRAME(3, 24),
-    ANIMCMD_FRAME(3, 24),
-    ANIMCMD_FRAME(0, 50),
+    ANIMCMD_FRAME(1, 31),
+    ANIMCMD_FRAME(4, 24),
+    ANIMCMD_FRAME(0, 1),
```

Comment:
I get that the other change was a merge error on expansion's part, but are these altered cmd frames also a merge error or are these your own changes?

Reply:
These were my own changes. I felt the one there already felt too delayed between the start of the point and the actual slide in of the mon, so I changed it to make it feel more like HGSS.

Reply:
Then I personally believe that we should follow merrp's branch, with these suggestions going to her branch and merrp being allowed to decide. The fix itself can still go to expansion, though.

Reply:
Fair enough, I'll move these changes

Reply:
Done